ClientApp/src/common/components/Types.tsx

```diff
@@ -17,6 +17,7 @@ export interface SelectedConnectionType {
     partitionKey: string;
     rowKey: string;
     phoneNumber: string;
+    matchedRecipientId: string;
```

Comment:
matchedRecipientId could be changed to matchedFamilyId

Reply:
This part of the code is used and fixed in another approved PR..
Removed this part of the code